docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>I'd think it's better to not have random variables in this example, so that people can eye-ball the correctness of the result. I'd just do <code>v = jnp.array([1.0, 2.0])</code> or something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Ah yes it's a good point, also because it's harder to read. Changed</p>